PLib/UI/PUIUtils.cs

```diff
@@ -74,39 +74,101 @@ public static class PUIUtils {
 		/// <param name="uiPrefab">The UI prefab to use. If null is passed, the UI should
 		/// be created and added to the GameObject hosting the controller object in its
 		/// OnPrefabInit function.</param>
-		public static void AddSideScreenContent<T>(GameObject uiPrefab = null)
+		/// <param name="inOrder">If the insertion should be performed in some order. The default of
+		/// false will simply insert at the end of the list which will make this screen appear
+		/// at the top of every side screen it applies to. Set to true to perform insertion
+		/// elsewhere in the list.</param>
+		/// <param name="insertBefore">If the insertion should be done before "insertionName" or
+		/// after "insertionName". True means before, false means after.</param>
+		/// <param name="insertionName">The name of the side screen that you want to choose
+		/// the ordering of this one around. Should be from this list:
+		/// Telepad Side Screen
```

Comment:
Consider using the class name of the target side screen to insert before, which improves compatibility if other mods declare side screens. It also allows the user to specify the screen with `nameof` on a strong ref to the side screen class.